spec/src/main/asciidoc/required-apis.asciidoc

```diff
@@ -59,8 +57,22 @@ MicroProfile 6.0 and above requires https://jakarta.ee/specifications/coreprofil
 * Jakarta JSON Processing 2.1
 * Jakarta RESTful Web Services 3.1
 
+Implementations based on the Jakarta EE 10 Core Profile must pass *all* required Jakarta EE 10 Core Profile TCKs.
+
 pass:[**] Full CDI support is not required. CDI SE support is not required.
 
+MicroProfile 6.0 allows optional support for Jakarta EE 9.1 and other Jakarta EE profiles provided the minimum APIs are implemented and *all* required TCK tests pass:
```

Comment:
I'm surprised to read this given the email thread on this subject concluded with the following:

> Thank you for the good discussion. _I think in the interest of moving things forward we can wrap this up as there being no real support for including MicroProfile 6.0 implementations based on Jakarta EE 9.1_. Not due to technical limitations, but due to preference, which is ok. 
https://groups.google.com/g/microprofile/c/ZngsfSv3RrQ

Reply:
See my note on the list.  Emily reached out me and there was another discussion on Tuesday where some willingness to revisit was expressed and I was requested to create a PR for people to evaluate.

Reply:
But I think there are technical problems.  A user can build an application with MP 6.0 API as the sole dependency and can use, for example the classes BeanContainer, BuildCompatibleExtension, EntityPart, etc..

The application builds fine but fails deployment as those classes are not available in a Jakarta EE 9.1 runtime.

So the compatibility no longer means it is able to run all applications build on top of the MP 6.0 API and thus certification s meaningless for vendors and users.

Reply:
@NottyCode it was the on the technical call.  I don't know where those are being posted these days.  @jclingan is that up anywhere?